core/domain/stats_jobs_one_off.py

```diff
@@ -227,6 +230,20 @@ def reduce(exp_id, stringified_values):
             elif value['event_type'] == feconf.EVENT_TYPE_STATE_HIT:
                 state_name = value['state_name']
                 session_id = value['session_id']
+                event_ts = value['created_on']
```

Comment:
Prefer event_timestamp or event_timestamp_msec (use the latter if it's msecs, use the former if it's datetime.datetime).

Reply:
Done.